integration-tests/avro-rpc/pom.xml

```diff
@@ -127,6 +127,7 @@
                     <plugin>
                         <groupId>org.apache.avro</groupId>
                         <artifactId>avro-maven-plugin</artifactId>
+                        <version>${avro-version}</version>
```

Comment:
I thought we have agreed to wrap Camel version props into our own `.version` properties in the top pom, so that we keep an overview in a single place which props we inherit from Camel. Maybe I am wrong remembering or interpreting what we agreed on :)

Reply:
Yes, you're right. I'll fix that.